### PR TITLE
Describe multiword route naming conventions.

### LINKED ISSUE #1404: Mention multi-word routes
Related: https://github.com/emberjs/ember.js/issues/11487

### PR DESCRIPTION
**Pull request by dpdawson**
closes #1404

### CODE REVIEW

File: source/localizable/routing/defining-your-routes.md [hunk 1]
```diff
@@ -46,6 +46,30 @@ routes, using the name that you provided to the `route` method.
...
+
+The route defined above will by default use the `blog-post.js` route handler,
+the `blog-post.hbs` template, and be referred to as `blog-post` in any
+`{{link-to}}` helpers.
+
+Multiword route names that break this convention, such as:
+
+```app/router.js
+Router.map(function() {
+  this.route('blog_post', { path: '/blog-post' });
```
Comment (locks) [inline source/localizable/routing/defining-your-routes.md]: didn't you mean it the other way around?

**Comment by alexspeller** [comment]
👍 this looks correct for me, although frankly I think it would be a good idea to just deprecate non-dasherized route names. Also, route names with a `/` in break stuff real bad (i.e. `this.route('foo/bar')`), it would be good to mention that somewhere, or deprecate and warn about it. But that's not a comment on this PR necessarily.

File: source/localizable/routing/defining-your-routes.md (hunk 1 above)
Comment (alexspeller) [inline source/localizable/routing/defining-your-routes.md]: I _think_ it's correct as written - module lookup for js file will be normalized, but not link-to

**Comment by rwjblue** [comment]
We should probably just deprecate if the first arg to `this.route` isn't already dasherized, but that is a story for another day 😝

**Comment by locks** [comment]
👌 

@homu r+

**Pull request merged**